docs/changes/modeling/14902.bugfix.rst

```diff
@@ -0,0 +1 @@
+All models except splines can be pickled now.
```

Comment:
Will splines be fixed in the future or it is simply impossible?

Reply:
I have not looked too much into pickling splines, just saw it's more work than the rest of the models. Given that no one is requesting it now I suggest we add this when it is needed or when someone has time to do it.